extensions/wikia/WikiaNewtalk/wikia_newtalk.php

```diff
@@ -77,11 +77,20 @@ function wfSetWikiaNewtalk( WikiPage $article ): bool {
         /**
 		 * then insert
 		 */
-		$dbw->insert(
-            "shared_newtalks",
-            wfBuildNewtalkWhereCondition( $other, true ),
-            __METHOD__
-        );
+	    $dbw->insert(
+		    "shared_newtalks",
+		    // TODO: bring back wfBuildNewtalkWhereCondition for anon as well, when the migration is completed (SUS-3089)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)